src/Grid/Col.jsx

```diff
@@ -0,0 +1,54 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PropTypes} from "react";
+
+import Size from "./Size";
+
+require("../less/grid.less");
+
+
+export default function Col({
+  children,
+  className,
+  span,
+  wrapperComponent: Wrapper,
+  ...additionalProps,
+}) {
+  const {cssClass} = Col;
+  let colSpanClasses;
+  if (typeof span === "number") {
+    colSpanClasses = `${cssClass.COL}--${Size.DEFAULT}--${span}`;
+  } else {
+    colSpanClasses = _.map(Size, s => span[s] && `${cssClass.COL}--${s}--${span[s]}`);
+  }
+
+  return (
+    <Wrapper className={classnames(cssClass.COL, colSpanClasses, className)} {...additionalProps}>
+      {children}
+    </Wrapper>
+  );
+}
+
+Col.colSpanPropShape = {};
+_.forEach(Size, size => {
+  Col.colSpanPropShape[size] = PropTypes.number;
+});
```

Comment:
sets up the prop type for the `span` prop. don't want to have to manually update it if we add a new `Size`, so setting it up dynamically.
added a comment to hopefully make it clearer